pkg/lokifrontend/frontend/v2/frontend_scheduler_worker.go

```diff
@@ -311,7 +311,7 @@ func (w *frontendSchedulerWorker) schedulerLoop(loop schedulerpb.SchedulerForFro
 				req.response <- &frontendv2pb.QueryResultRequest{
 					HttpResponse: &httpgrpc.HTTPResponse{
 						Code: http.StatusInternalServerError,
-						Body: []byte(err.Error()),
+						Body: []byte(resp.Error),
```

Comment:
would be nice if could lock this behaviour in any of the tests but non-blocker for this PR.